examples/archive.html

```diff
@@ -104,17 +111,20 @@ <h3 id="offcanvasRightLabel">Images</h3>
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
+                let imageArray=[];
                 placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
                 placeButton.innerHTML = 'Place on map';
-
+                let imageGroup = L.distortableCollection().addTo(map)
                 image.src = `${url.replace('metadata', 'download')}/${file.name}`;
-
                 imageRow.classList.add('d-flex', 'justify-content-between', 'align-items-center', 'mb-4', 'pe-5');
                 imageRow.append(image, placeButton);
-
+                imageArray.push[image]
```

Comment:
Should we do this only when the user presses the Place button? Maybe in the click handler down on line 141? https://github.com/publiclab/Leaflet.DistortableImage/pull/1161/files#diff-4e935909ea43d3a9c21496afdeae5865a74c6d7ee77640cf467b6e3911b6c63dL141